components/widgets/menu/menu_header.scss

```diff
@@ -6,15 +6,15 @@
         clear: both;
         color: inherit;
         cursor: default;
-        display: block;
+        display: flex;
         font-weight: normal;
         padding: 0 32px 0 24px;
         text-align: left;
         white-space: nowrap;
         width: 100%;
         overflow: hidden;
         text-overflow: ellipsis;
-        line-height: 28px;
         font-weight: $font-weight--semibold;
+        height: 36px;
```

Comment:
same here

Reply:
@michelengelen These are global changes that need to be represented for all menu headers.
We ideally don't want different menu header sizes in the app. So we should be consistent.

You can see the global figma component here:
https://www.figma.com/file/tbMvXozL5f8Uc8oWYTEzbh/Patterns-Popover-Menus?node-id=532%3A3586